automation/taskcluster/lib/tasks.py

```diff
@@ -185,6 +185,35 @@ def _craft_clean_gradle_task(
             treeherder=treeherder,
         )
 
+    def craft_dependencies_task(self):
+        # Output the dependencies to an artifact.  This is used by the
+        # telemetry probe scraper to determine all of the metrics that
+        # &browser might send (both from itself and any of its dependent
+        # libraries that use Glean).
+        return self._craft_clean_gradle_task(
+            name='dependencies',
+            description='Write dependencies to a build artifact',
+            gradle_task='app:dependencies --configuration implementation > dependencies.txt',
```

Comment:
This kind of depends on an unsanitized string interpolation.  Consider making this an actual Gradle task that writes the file, so you don't abuse the interpolation.